protobuf/transform.go

```diff
@@ -128,6 +144,21 @@ func (t *Transformer) transformFunc(pkg *Package, f *scanner.Func, names nameSet
 	return rpc
 }
 
+func (t *Transformer) transformInterface(pkg *Package, f *scanner.Interface, names nameSet) *Service {
+	srv := &Service{
+		Name: f.Name,
+	}
+	for _, m := range f.Methods {
+		if rpc := t.transformFunc(pkg, m, names); rpc != nil {
+			srv.RPCs = append(srv.RPCs, rpc)
+		}
+	}
+	if len(srv.RPCs) == 0 {
+		return nil
```

Comment:
There is no test checking what happens if we generate an empty interface. Can we add one to make sure nothing explodes?